CairoMakie/src/utils.jl

```diff
@@ -120,10 +120,14 @@ function numbers_to_colors(numbers::AbstractArray{<:Number}, primitive)
         colorrange = extrema(numbers)
     end
 
-    Makie.interpolated_getindex.(
+    raw_colors = Makie.interpolated_getindex.(
         Ref(colormap),
         Float64.(numbers), # ints don't work in interpolated_getindex
         Ref(colorrange))
+
+    !isnothing(to_value(get(primitive, :lowclip,  nothing))) && (raw_colors[numbers .< colorrange[1]] .= Makie.to_color(primitive.lowclip[]))
+    !isnothing(to_value(get(primitive, :highclip, nothing))) && (raw_colors[numbers .< colorrange[2]] .= Makie.to_color(primitive.highclip[]))
```

Comment:
This should be `>`, right?

Reply:
whoops, thanks!  guess this is what I get for coding on a plane :D

Reply:
```suggestion
    !isnothing(to_value(get(primitive, :highclip, nothing))) && (raw_colors[numbers .> colorrange[2]] .= Makie.to_color(primitive.highclip[]))